experimental/gradually-typed/examples/neural-interpreter/STLC.hs

```diff
@@ -93,15 +95,15 @@ instance ToJSON a => ToJSON (Scope () Exp a)
 
 instance ToJSON a => ToJSON (Exp a)
 
-deriveEq1 ''Exp
-deriveShow1 ''Exp
-deriveOrd1 ''Exp
-
-instance Eq a => Eq (Exp a) where (==) = eq1
-
-instance Ord a => Ord (Exp a) where compare = compare1
-
-instance Show a => Show (Exp a) where showsPrec = showsPrec1
+fmap concat $ sequence
+  [ deriveEq1   ''Exp
+  , deriveOrd1  ''Exp
+  , deriveShow1 ''Exp
+  , [d| instance Eq a => Eq (Exp a) where (==) = eq1
+        instance Ord a => Ord (Exp a) where compare = compare1
+        instance Show a => Show (Exp a) where showsPrec = showsPrec1
+      |]
+  ]
```

Comment:
It is modified in the same way as the bound package.
https://github.com/ekmett/bound/blob/master/examples/Simple.hs#L64-L74